src/appleseed/renderer/modeling/scene/objectinstance.cpp

```diff
@@ -459,7 +459,11 @@ bool ObjectInstance::on_frame_begin(
 
     const EntityDefMessageContext context("object instance", this);
 
-    if (uses_alpha_mapping())
+    const bool uses_materials_alpha_mapping =
```

Comment:
When you use an object alpha map but you have different front and back materials you get this warning. I think that in this case, the result is correct (the object alpha map is used in both sides) 
and the warning is wrong.